### ISSUE TITLE
Update Search API to include searrching of synonyms

### ISSUE THREAD

**Issue by gnewton** [AAFC-MBB/Host-PathogenDB#2]
The search API does not now allow for searching for synonyms (dependent on #1 ). Extend API implementation to support synonym search.

**Comment by gnewton** [comment]
I am not sure why this is closed: there are no commits against this issue. (I was getting to it...)

**Comment by KTCloud** [comment]
Sorry I thought I add the commit the change! I will be updating tonight or tomorrow.
